src/App.js

```diff
@@ -0,0 +1,101 @@
+import { useEffect, useState } from 'react'
+import { Routes, Route, createSearchParams, useSearchParams, useNavigate } from "react-router-dom"
+import { useDispatch, useSelector } from 'react-redux'
+import 'reactjs-popup/dist/index.css'
+import { fetchMovies } from './data/moviesSlice'
+import { ENDPOINT_SEARCH, ENDPOINT_DISCOVER, ENDPOINT, API_KEY } from './constants'
+import Header from './components/Header'
+import Movies from './components/Movies'
+import Starred from './components/Starred'
+import WatchLater from './components/WatchLater'
+import YouTubePlayer from './components/YoutubePlayer'
+import './app.scss'
```

Comment:
I would recommend to use the styles a little bit different, 
```suggestion
import styles from './app.scss'
```
this way we can refer to classes like
` return (
    <div className={styles.App}>
      />
`
This way is cleaner and let us control the styles like variables instead of plain text. If we use css module the scope will be restricted to the component